engine/dlib/src/test/test_httpclient.cpp

```diff
@@ -800,6 +803,8 @@ TEST_P(dmHttpClientTest, Cache)
 
 TEST_P(dmHttpClientTest, MaxAgeCache)
 {
+    signal(SIGPIPE, SIG_IGN);
```

Comment:
Note: This doesn't run on linux without suppressing the sigpipe. I have no idea why the tests behave this way, but if you run the tests in the normal gdb you will still get the sigpipe, so I don't think the build system upgrade is at fault. Also, it only seems to happen on CI, if I run this test on a ubuntu VM the test passes, eventhough it fails in gdb..